frontend/website/src/Testnet.re

```diff
@@ -7,8 +7,70 @@ let extraHeaders = () =>
       integrity="sha256-0Ed5s/n37LIeAWApZmZUhY9icm932KvYkTVdJzUBiI4="
       crossOrigin="anonymous"
     />
+    <script src={Links.Cdn.url("/static/js/termynal.js")} />
+    <link rel="stylesheet" href={Links.Cdn.url("/static/css/termynal.css")} />
   </>;
 
+module Icon = {
```

Comment:
Is this equal to `GetInvolvedSection.SocialLink.Svg.discord`?

Maybe we can either use that reference or if we don't want to reach into that page, pull out the `SocialLink.Svg` into its own file or something?

Reply:
added to a TODO